client/game/highScoreMode.ts

```diff
@@ -1,8 +1,14 @@
 import { IGameState } from "../comets";
 import { Highscores } from "./highscores";
 import { Screen } from "./screen";
+import { isAddress } from "viem";
+
 const formatAddress = (address: string) => {
-  return `${address.slice(0, 6)}...${address.slice(-4)}`;
+  if (isAddress(address)) {
+    return `${address.slice(0, 6)}...${address.slice(-4)}`;
+  }
+  // Is an ENS name then
+  return address.padEnd(14, " ");
```

Comment:
do we need this? we pad it in UI anyways

Reply:
ens names need extra padding because they dont fill up the full space

Reply:
We definitely give more than address padding there, it should work imo